solutions/fully-configurable/README.md

```diff
@@ -0,0 +1,13 @@
+ # IBM Cloud Databases for MySQL
+
+This architecture creates an instance of IBM Cloud Databases for MySQL and supports provisioning of the following resources:
+
+- A resource group, if one is not passed in.
```

Comment:
do we still create resource group?
all bullets must be updated (removed if not true or add any new) accordingly.